app/api/ada/lib/storage/bridge/updateTransactions.js

```diff
@@ -317,6 +317,45 @@ export async function rawGetTransactions(
         *    Again, it's easier to say it's just whoever gets it
       */
       ownImplicitInput: new BigNumber(0),
+      ownImplicitOutput: (() => {
+        if (tx.txType === TransactionType.CardanoShelley) {
+          let implicitOutputSum = new BigNumber(0);
+          for (const cert of tx.certificates) {
+            if (
+              cert.certificate.Kind !==
+              RustModule.WalletV4.CertificateKind.MoveInstantaneousRewardsCert
+            ) {
+              continue;
+            }
+            // TODO: uncomment once this works in the WASM side
```

Comment:
This is unfortunate we can't have this, but we can live without it until Rob can look into it

Reply:
what this does?